source/slang/parser.cpp

```diff
@@ -3718,7 +3725,7 @@ namespace Slang
         {
         case TokenType::QuestionMark:
             opToken = parser->ReadToken();
-            opToken.Content = "?:";
+            opToken.Content = UnownedStringSlice::fromLiteral("?:");
```

Comment:
Hmm... Here we are doing the exact same thing again, so maybe I was wrong about the earlier case of adding a `:` - one must be at the declaration site and one at the expression parsing site. They should both be able to use the simple logic that this case employs.